libpod/container.go

```diff
@@ -146,15 +146,11 @@ type containerState struct {
 	// IPs contains IP addresses assigned to the container
```

Comment:
Trim this comment, we killed the IP field

Reply:
done